### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by Bondok6**
Make the project display correctly on desktop screens using **media queries**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -275,6 +286,7 @@ <h3 class="card__text card__text--about">Skills</h3>
         </ul>
       </section>
     </main>
+
     <section class="contact">
 
       <h3 class="title">
```
Comment (MrBrN197) [inline index.html]: - [x] You've done really well with your design 👍 It matches the Figma template really well; one thing I'd like you to notice, however, is that the form area is supposed to be aligned with the main `.hero` section's content in the Figma template. Please address this issue.
## Your Design
image
## Figma Design
image

File: styles/main.css [hunk 2]
```diff
@@ -99,6 +95,17 @@ li {
...
+  }
+
+  .links {
+    width: 20vw;
+  }
+}
+
 /* End General Rules */
 
 /* Start Header Section */
```
Comment (MrBrN197) [inline styles/main.css:111]: - [ ] The rest of your design matches extremely well. Please add a `background-color` to the `header` element though; right now it's transparent and causes things behind the header to be visible under the text of the header which is visually distracting. 
- [x] Or you could make sure that the header stays at the top of the page and doesn't stay fixed on the page.
## Issue (transparent background)
image

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @Bondok6 👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔Design matches the Figma really well.
- ✔Professional README
- ✔Organized clean code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi @Bondok6 👋, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**